src/components/footer/Footer.module.css

```diff
@@ -0,0 +1,8 @@
+.icon:hover {
```

Comment:
Can we leverage the `_hover` attribute on the link? This will help to reduce the need for an extra stylesheet.

Reply:
Ahh amazing! I didn't know about that. Glad I can get rid of that css file